docs/courses-vscode-extension.md

```diff
@@ -4,7 +4,7 @@ This details the maintenance guidelines for the [freeCodeCamp/courses-vscode-ext
 
 ## Publishing the Extension
 
-A GitHub Action automagically publishes the extension to the Visual Studio Marketplace, on the release of a new GitHub Release.
+A GitHub Action automatically publishes the extension to the Visual Studio Marketplace, on the release of a new GitHub Release.
```

Comment:
This one is actually intentional.
automagically is a word 
https://www.dictionary.com/browse/automagically

revert this back to the way it was

```suggestion
A GitHub Action automagically publishes the extension to the Visual Studio Marketplace, on the release of a new GitHub Release.
```